README.md

```diff
@@ -41,3 +43,26 @@ make demo
 
 The demo installation uses Helm, and roughly matches what you'd get by installing trust-manager into your own
 cluster using Helm - although it uses locally-built images rather than the ones we publish publicly.
+
+## Example Bundle
+
+The simplest useful Bundle to start with is likely to be one using default CAs, which are available from trust-manager 0.4.0+.
+
+This default CA package is based on Debian's `ca-certificates` package, and so matches what you'd expect to see in a Debian
+container or VM.
+
+```yaml
+apiVersion: trust.cert-manager.io/v1alpha1
+kind: Bundle
+metadata:
+  name: trust-manager-bundle
+spec:
+  sources:
+  - useDefaultCAs: true
+  target:
+    configMap:
+      key: "bundle.pem"
+```
+
+This Bundle will lead to a ConfigMap called `trust-manager-bundle` being created in all namespaces, ready to be mounted
+and used by your applications. For more details see the [trust-manager documentation](https://cert-manager.io/docs/projects/trust-manager/).
```

Comment:
Suggestion: `This Bundle will lead to a `trust-manager-bundle` ConfigMap containing the default CAs being created in all namespaces and continuously kept up to date if the default CAs change...`

I think it's worth to outline that trust-manager continously ensures that target CA resources are up to date as sources change as opposed to doing a one time thing- I've seen that folks who are less familiar with Kubernetes need this explicitly said wrt cert-manager certificate renewal.

Reply:
Thanks, I like this suggestion!

This PR already merged before I saw your comment; I raised https://github.com/cert-manager/trust-manager/pull/111 - could you review that? :grin: